x-pack/plugins/cases/server/client/cases/validators.ts

```diff
@@ -133,3 +137,49 @@ export const validateRequiredCustomFields = ({
     );
   }
 };
+
+export const validateSearchCasesCustomFields = ({
+  customFieldsConfiguration,
+  customFields,
+}: {
+  customFieldsConfiguration: CustomFieldsConfiguration;
+  customFields: CasesSearchRequest['customFields'];
+}) => {
+  let customFieldsMapping: ICasesCustomField | null | undefined = null;
```

Comment:
what's this `undefined = null`?

Reply:
Fixed in https://github.com/elastic/kibana/pull/170297/commits/1c993c7d782d1ecf8c6ec8a7e53a0569ba43383d